shell/initialize/client.js

```diff
@@ -215,13 +204,14 @@ function callMiddleware(Components, context, layout) {
     if (layout.options.middleware) {
       midd = midd.concat(layout.options.middleware);
     }
-    Components.forEach((Component) => {
-      if (Component.options.middleware) {
-        midd = midd.concat(Component.options.middleware);
-      }
-    });
   }
```

Comment:
Will `layout` ever be defined here?

Reply:
It won't. I removed this.layout entirely. Removing this as well as the layout argument from middleware.